presto-jdbc/src/test/java/io/prestosql/jdbc/TestPrestoDriverUri.java

```diff
@@ -68,17 +77,17 @@ public void testInvalidUrls()
         assertInvalid("jdbc:presto://localhost:8080/hive/default?SSL=true&SSLVerification=", "Connection property 'SSLVerification' value is empty");
 
         // property in url multiple times
-        assertInvalid("presto://localhost:8080/blackhole?password=a&password=b", "Connection property 'password' is in URL multiple times");
```

Comment:
Please add a test case verifying "presto://..." is not a valid URL
(apparently before your change it was parsed successfully)